### PR TITLE
FEATURE: NAS-232 Add explainAfm download resource for tiger backend

### PR DESCRIPTION
**Pull request by Tulean**
JIRA: NAS-232

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by Tulean** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1116/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4807/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1116/ : SUCCESS in 31m 33s

File: libs/api-client-tiger/src/generated/metadata-json-api/base.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { Configuration } from "./configuration";
 // @ts-ignore
 import globalAxios, { AxiosPromise, AxiosInstance } from "axios";
 
-export const BASE_PATH = "http://localhost".replace(/\/+$/, "");
+export const BASE_PATH = "https://staging.anywhere.gooddata.com".replace(/\/+$/, "");
```
Comment (no23reason) [inline libs/api-client-tiger/src/generated/metadata-json-api/base.ts:21]: This is suspicious, why did this change?

File: libs/api-client-tiger/src/layout.ts [hunk 2]
```diff
@@ -1,10 +1,6 @@
...
-import {
-    DeclarativeLayoutControllerApi,
-    DeclarativeLayoutControllerApiInterface,
-} from "./generated/metadata-json-api";
+import { LayoutApi, LayoutApiInterface } from "./generated/metadata-json-api";
 
-export const tigerDeclarativeLayoutClientFactory = (
-    axios: AxiosInstance,
-): DeclarativeLayoutControllerApiInterface => new DeclarativeLayoutControllerApi({}, "", axios);
+export const tigerDeclarativeLayoutClientFactory = (axios: AxiosInstance): LayoutApiInterface =>
```
Comment (no23reason) [inline libs/api-client-tiger/src/layout.ts:5]: Nitpick: this should probably now be named `tigerLayoutClientFactory`.

**Review by no23reason — CHANGES_REQUESTED** [review]
We should probably add this as a backendCapability, something like `supportsExplainAfm` and set it properly. This would allow us to check for it in the apps and not call it if it is not supported.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4809/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6449/ : FAILURE in 6m 13s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4811/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 18s
- gooddata-ui-sdk-unit-tests : SKIPPED

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4813/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6451/ : SUCCESS in 19m 11s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1627

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1361/ : SUCCESS in 1m 02s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1637/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/421/ : SUCCESS in 11m 53s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6455/ : SUCCESS in 14m 23s

**Pull request merged**